types/openid-client/index.d.ts

```diff
@@ -89,6 +90,22 @@ export interface EndSessionUrlParameters {
     readonly state?: string;
 }
 
+export interface IntrospectionResponse {
+    active: boolean;
+    scope?: string[];
```

Comment:
Why an array?
![image](https://user-images.githubusercontent.com/388796/62368039-abad6700-b52b-11e9-83ce-842a33891918.png)

Reply:
Not quite sure why I thought this was an array, I've updated the type to be a string